proxy/http/remap/UrlRewrite.cc

```diff
@@ -83,6 +83,14 @@ UrlRewrite::load()
   /* Initialize the plugin factory */
   pluginFactory.setRuntimeDir(RecConfigReadRuntimeDir()).addSearchDir(RecConfigReadPluginDir());
 
+  /* Initialize the next hop strategy factory */
+  std::string sf = RecConfigReadConfigPath("proxy.config.url_remap.strategies.filename", "strategies.yaml");
+  Debug("url_rewrite_regex", "strategyFactory file: %s", sf.c_str());
+  strategyFactory = new NextHopStrategyFactory(sf.c_str());
+  if (!strategyFactory->strategies_loaded) {
+    Warning("no strategies were loaded, check your configuration and the error log.");
```

Comment:
@gtenev I was thinking that this would eventually replace ParentSelection but until then I could remove this warning message until such time as this does replace ParentSelection.

Reply:
it may be useful warning with or w/o the `ParentSelection`, my point was more not to ask the operator to look for problems if it is normal not to have any strategy defined. Looked around before I made this comment and it seems that the code will diligently error in case of real errors during the configuration parsing (as much as I can see just by reading the code)

Reply:
@gtenev I'll remove the warning.